FHIR/src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Visitors/AnonymizationVisitor.cs

```diff
@@ -67,6 +68,9 @@ public override void EndVisit(ElementNode node)
 
         private ProcessResult ProcessResourceNode(ElementNode node)
         {
+            // Initialize cache for every resource node
```

Comment:
consider to use ‘IAnonymizerProcessor’ interface

Reply:
Sure. ResourceProcessor is added in the update.